pipelines/folioRancher/folioNamespaceTools/manageNamespace/createDailySnapshotECS/Jenkinsfile

```diff
@@ -9,9 +9,9 @@ import org.folio.rest_v2.Users
 import org.folio.utilities.Tools
 import org.jenkinsci.plugins.workflow.libs.Library
 
-@Library('pipelines-shared-library') _
+@Library('pipelines-shared-library@RANCHER-1517') _
```

Comment:
Please do not forget to change before merge

Reply:
> Please do not forget to change before merge

There is an active build: https://jenkins-aws.indexdata.com/view/KitFox/job/folioRancher/job/folioNamespaceTools/job/createDailySnapshotECS/457/console
Once completed required switch to master branch will be in the place. No worries.